csharp/Platform.IO/FileHelpers.cs

```diff
@@ -20,6 +52,22 @@ public static T[] ReadAll<T>(string path)
             return reader.ReadAll<T>();
         }
 
+        /// <summary>
+        /// <para>Reads and returns the <typeparamref name="T"/> structure value from the file at the <paramref name="path"/>.</para>
```

Comment:
`the <typeparamref name="T"/> structure value` ⇾ `the first <typeparamref name="T"/> structure value`